server/src/main/kotlin/core/adapters/PlayerRouter.kt

```diff
@@ -0,0 +1,14 @@
+package core.adapters
+
+import core.bettinground.ActionValidation
+import core.bettinground.BettingAction
+import core.bettinground.Fold
+import core.gameflow.handstate.HandState
+
+class PlayerRouter {
+    fun getAction(playerId: Int): BettingAction = Fold
+    fun sendPrivateUpdate(playerId: Int, actionValidation: ActionValidation) {}
+    fun sendPrivateUpdate(playerId: Int, handState: HandState) {}
```

Comment:
Why would we ever want to send an update of the hand state secretly to only one player?